Browser/keywords/browser_control.py

```diff
@@ -48,8 +48,9 @@ def go_to(self, url: str, timeout: Optional[timedelta] = None):
         """Navigates to the given ``url``.
 
         ``url`` <str> URL to be navigated to.
+        
```

Comment:
Most likely here is space, at least flake8 says 
`Browser/keywords/browser_control.py:51:1: W293 blank line contains whitespace`
Could you trim the space away.

Reply:
Done :-)